src/components/dialog-full-screen/dialog-full-screen.js

```diff
@@ -91,16 +103,44 @@ class DialogFullScreen extends Modal {
   }
 
   /**
-   * Returns title prop wrapped in <h2> if title is a string otherwise returns the title prop directly.
-   * Dialog main classes.
+   * Overrides the original function to disable the document's scroll.
+   */
+  get onOpening() {
+    Browser.addClass(this.document.body, DIALOG_OPEN_BODY_CLASS);
```

Comment:
the data attribute wouldn't trigger the CSS redraw properly in IE which is why I updated it to use a class instead (I guess using a class IE is more liberal with CSS updates).

Keeping the helper function allows us to check if the class already exists more easily so we don't get duplicate classes.